### PR TITLE
kdbg: init at 2.5.5

### PR DESCRIPTION
**Pull request by eduarrrd**
It works in terms of functionality. However, I don't think I'm wrapping the right things since icons are missing and the open dialog pops modal dialogs about missing mimetypes. Probably a ksycoca issue? I tried looking into wrapKDEProgram but that is defined only in the docs and does not appear in the repository from what I can tell.

@ttuegel Could you take a look at this and point at what wrappers I'm missing?

### CODE REVIEW

**Comment by ttuegel** [comment]
There are no wrappers for KDE 4. For the missing icons and mimetypes, you could wrap the program to set `XDG_DATA_DIRS` to point at the appropriate packages.

File: pkgs/development/tools/misc/kdbg/default.nix [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+  meta = with stdenv.lib; {
+    homepage = http://www.kdbg.org/;
+    description = ''
+      A graphical user interface to gdb, the GNU debugger. It provides an
+      intuitive interface for setting breakpoints, inspecting variables, and
+      stepping through code.
+    '';
+    platform = unix;
+    license = license.gpl2;
```
Comment (domenkozar) [inline pkgs/development/tools/misc/kdbg/default.nix]: `license` attribute doesn't exist

**Comment by joachifm** [comment]
@eduarrrd ping

**Comment by garbas** [comment]
@eduarrrd ping

**Pull request closed without merging**